### PR TITLE
Correct css.types.counters

### PR DESCRIPTION
**Pull request by Elchi3**
The export was wrong here (`attr` instead of `counters`)

Real values are based on caniuse https://caniuse.com/#feat=css-counters

### CODE REVIEW

File: css/types/counters.json [hunk 1]
```diff
@@ -25,22 +25,22 @@
...
-              "version_added": true
+              "version_added": "10"
             },
             "opera_android": {
-              "version_added": true
+              "version_added": "10.1"
             },
             "safari": {
-              "version_added": true
+              "version_added": "3.1"
```
Comment (ddbeck) [inline css/types/counters.json]: I spot checked this and found https://trac.webkit.org/changeset/16721/webkit the changes of which are in the 3.0 release https://trac.webkit.org/browser/webkit/releases/Apple/Safari%203.0/WebCore/css/Counter.h

```suggestion
              "version_added": "3"
```

File: css/types/counters.json [hunk 2]
```diff
@@ -25,22 +25,22 @@
...
             },
             "safari_ios": {
-              "version_added": true
+              "version_added": "3.1"
```
Comment (ddbeck) [inline css/types/counters.json]: And if the other Safari data is true, then this should be 1

```suggestion
              "version_added": "1"
```

File: css/types/counters.json [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
     "types": {
-      "attr": {
+      "counters": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/counters",
           "description": "<code>counters()</code>",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "4"
```
Comment (ddbeck) [inline css/types/counters.json]: Again, following the WebKit change:

```suggestion
              "version_added": "1"
```

**Comment by Elchi3** [comment]
Nice finding! Applied your suggestions :+1:

**Pull request merged**